client/src/components/AutocompleteText.js

```diff
@@ -0,0 +1,57 @@
+import React, {Component} from 'react';
+import '../css/AutocompleteText.css';
+
+export default class AutocompleteText extends Component {
+    constructor(props){
+        super(props)
+        this.state = {
+            suggestions : [],
+            filter : '',
+            text : ''
+        }
+    }
+    
+    onTextChanged = (e) =>{
+        const { items } = this.props;
+        const value = e.target.value;
+        let suggestions = [];
+        if(value.length > 0){
+            const regex = new RegExp(`^${value}`,'i');
+            suggestions = items.sort().filter(v => regex.test(v)).splice(0,10);
+        }
+        this.setState(() => ({suggestions, text : value}));
```

Comment:
Suggestions data are loaded only once when the app started. You get the autocomplete results based on the previously fetched suggestions by filtering it using the search text value.

What if there is a new data loaded in the db. Since the suggestions were already loaded and it is loaded only once, the new data are not added to the autocomplete/suggestion results.

How do you deal with this problem?